odbc/src/api/get_info_tests.rs

```diff
@@ -0,0 +1,633 @@
+        expected_sql_return = SqlReturn::SUCCESS,
+        buffer_length = 11,
+        expected_length = 10,
+        expected_value = "00.01.0000",
+        actual_value_modifier = modify_string_value,
+    );
+
+    test_get_info!(
+        driver_odbc_ver,
+        info_type = InfoType::DriverOdbcVer,
+        expected_sql_return = SqlReturn::SUCCESS,
+        buffer_length = 6,
+        expected_length = 5,
+        expected_value = "03.08",
+        actual_value_modifier = modify_string_value,
+    );
+
+    test_get_info!(
+        search_pattern_escape,
+        info_type = InfoType::SearchPatternEscape,
+        expected_sql_return = SqlReturn::SUCCESS,
+        buffer_length = 1,
+        expected_length = 0,
+        expected_value = "",
+        actual_value_modifier = modify_string_value,
+    );
+
+    test_get_info!(
+        dbms_name,
+        info_type = InfoType::DbmsName,
+        expected_sql_return = SqlReturn::SUCCESS,
+        buffer_length = 14,
+        expected_length = 13,
+        expected_value = "MongoDB Atlas",
+        actual_value_modifier = modify_string_value,
+    );
+
+    // DbmsVer must be an integration test since it must connect to ADL to get the version
+
+    test_get_info!(
+        concat_null_behavior,
+        info_type = InfoType::ConcatNullBehavior,
+        expected_sql_return = SqlReturn::SUCCESS,
+        expected_length = std::mem::size_of::<u16>() as i16,
+        expected_value = 0u16,
+        actual_value_modifier = modify_u16_value,
+    );
+
+    test_get_info!(
+        identifier_quote_char,
+        info_type = InfoType::IdentifierQuoteChar,
+        expected_sql_return = SqlReturn::SUCCESS,
+        buffer_length = 2,
+        expected_length = 1,
+        expected_value = "`",
+        actual_value_modifier = modify_string_value,
+    );
+
+    test_get_info!(
+        owner_term,
+        info_type = InfoType::OwnerTerm,
+        expected_sql_return = SqlReturn::SUCCESS,
+        buffer_length = 1,
+        expected_length = 0,
+        expected_value = "",
+        actual_value_modifier = modify_string_value,
+    );
+
+    test_get_info!(
+        catalog_name_separator,
+        info_type = InfoType::CatalogNameSeparator,
+        expected_sql_return = SqlReturn::SUCCESS,
+        buffer_length = 2,
+        expected_length = 1,
+        expected_value = ".",
+        actual_value_modifier = modify_string_value,
+    );
+
+    test_get_info!(
+        catalog_term,
+        info_type = InfoType::CatalogTerm,
+        expected_sql_return = SqlReturn::SUCCESS,
+        buffer_length = 9,
+        expected_length = 8,
+        expected_value = "database",
+        actual_value_modifier = modify_string_value,
+    );
+
+    test_get_info!(
+        convert_functions,
+        info_type = InfoType::ConvertFunctions,
+        expected_sql_return = SqlReturn::SUCCESS,
+        expected_length = std::mem::size_of::<u32>() as i16,
+        expected_value = 2u32,
```

Comment:
Not a real big deal, but it's hard to tell if the bit flags are correct this way. I'm not saying the value returned itself, I fully trust you checked it, but more the values used to get to this.
Could you express them similarly to how they are defined in sql_get_info?

Reply:
done